dockerbuild/accelerator_docker_build.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+set -euxo pipefail
+
+dockerimage="debian:11-slim"
+
+
+# we do this so that we can be agnostic about where we're invoked from
+# meaning you can exec this script anywhere and it should work the same
+thisiswhereiam=${BASH_SOURCE[0]}
+# this should be /whatever/directory/structure/Open-Fortress-Source
```

Comment:
It should be?

Reply:
Ditto. @sapphonie

Reply:
Sorry, copied this build script from myself somewhere else 👀 , the directory structure shouldn't matter

You can yeet the entire `dockerbuild/` folder if you want, it's not really needed with proper GH actions

Reply:
Noted! I'll rebase this one last time.